flink-contrib/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBMapState.java

```diff
@@ -319,6 +321,9 @@ private UV deserializeUserValue(byte[] rawValueBytes) throws IOException {
 		/** The raw bytes of the value stored in RocksDB. */
 		private byte[] rawValueBytes;
 
+		/** The offset of User Key offset in raw key bytes. */
+		private final int userKeyOffset;
```

Comment:
This value is basically a constant on a per-backend level and we could remember the value in the MapState itself, no need to replicate it into every entry. It can then still be recalled from there for `deserializeUserKey(...)`.

Reply:
👍 addressed.